composer.json

```diff
@@ -88,7 +88,6 @@
         "drupal/migrate_tools": "8.2.0-beta1",
         "drupal/oauth": "8.1.x-dev",
         "drupal/page_manager": "8.1.0-alpha23",
-        "drupal/panels": "8.3.x-dev",
```

Comment:
Yeah - we have a panels mega patch that includes 20 some commits. See [What's up with the 710k patch to panels?](http://lightning.acquia.com/blog/whats-up-700k-patch-to-panels) :)